sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/Ast.scala

```diff
@@ -1283,7 +1283,6 @@ object Ast {
         metadata: PackageMetadata,
     ): GenPackage[E] = {
       val isUtilityPackage =
-        metadata.name == "daml-prim" || metadata.name == "daml-stdlib" ||
```

Comment:
Oh, so prim and stdlib don't export anything serializable anymore? Did we update them on the 3x branch?

Reply:
I noticed later on you updated RoundingMode to not be serializable, so I guess the answer to this is yes.